Common/util/mutifilelib.cpp

```diff
@@ -112,14 +112,14 @@ MFLUtil::MFLError MFLUtil::ReadSigsAndVersion(Stream *in, int *p_lib_version, lo
     if (HeadSig.Compare(sig) != 0)
     {
         // signature not found, check signature at the end of file
-        in->Seek(-TailSig.GetLength(), kSeekEnd);
+        in->Seek(-(int)TailSig.GetLength(), kSeekEnd);
```

Comment:
TailSig is a very short constant string (around 10 characters), if it is what you are refering to?